docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -331,3 +331,36 @@ Used to lookup fungible assets balances
 
 * `202 Accepted` - the data for the address is not fulfilled yet, re-trying is needed in up to 180 seconds.
 * `400 Bad request` - wrong requested arguments format.
+
+## Fungibles
+
+### Price
+
+Lookup for the current price of the fungible assset item:
+
+`GET /v1/fungibles/price`
+
+#### Path parameters
+
+* `projectId` - The project identifier.
+* `currency` - Currency for the price calculation e.g. `usd`.
+    * Supported currencies: `usd`, `eur`, `gbp`, `aud`, `cad`, `inr`, `jpy`, `btc`, `eth`.
+* `address` - Asset's contract address in [CAIP10](https://github.com/ChainAgnostic/CAIPs/blob/main/CAIPs/caip-10.md) format.
```

Comment:
@enesozturk is the address useful for you? 
Wouldn't it be simpler to send a symbol?

Requiring an address here would require the ui to somehow have access to it which will probably be an extra integration on FE. 

Wondering if chainid + symbol would be better

Reply:
@tomiir I think the address is required by the API we are using. We send address of the tokens to fetch prices. I'm not sure if we are using 1Inch API or Zerion. In 1Inch price API, the prices endpoint gets address array instead of one address and returns the array as response.

We need this features in swaps to fetch our owned tokens balances at once. @geekbrother